vnext/Shared/OInstance.cpp

```diff
@@ -430,6 +435,23 @@ InstanceImpl::InstanceImpl(
     if (m_devSettings->jsiRuntimeHolder) {
       jsef = std::make_shared<OJSIExecutorFactory>(
           m_devSettings->jsiRuntimeHolder, m_devSettings->loggingCallback);
+    } else if (m_devSettings->jsiEngineOverride != JSIEngineOverride::Default) {
+      switch (m_devSettings->jsiEngineOverride) {
```

Comment:
>s [](start = 6, length = 1)

one more indentation level?

Reply:
Unfortunately, for some reason we enforce the formatting with clang-format and actually fail the PRs for that reason.

---
In reply to: [314904875](https://github.com/microsoft/react-native-windows/pull/2944#discussion_r314904875) [](ancestors = 314904875)

Reply:
There is a macro to turn clang format off for certain sections. We should only enforce it when it makes sense.

---
In reply to: [314905445](https://github.com/microsoft/react-native-windows/pull/2944#discussion_r314905445) [](ancestors = 314905445,314904875)